CI:  UT TestCooldownAtimeOnWriteFixedOnLoad failed.

https://github.com/juicedata/juicefs/actions/runs/21681326825/job/62516313283

```
=== RUN   TestCooldownAtimeOnWriteFixedOnLoad
2026/02/04 17:30:56.226491 juicefs[32579] <WARNING>: unknown cache eviction policy: "", fallback to 2-random [newCacheStore@disk_cache.go:118]
2026/02/04 17:30:56.226570 juicefs[32579] <INFO>: Disk cache (/tmp/TestCooldownAtimeOnWriteFixedOnLoad2954228371/001): used ratio - [space 30.4%, inode 4.7%] [newCacheStore@disk_cache.go:153]

  mock time.Now to avoid drift     disk_cache_test.go:643: 
        	Error Trace:	/home/runner/work/juicefs/juicefs/pkg/chunk/disk_cache_test.go:643
        	            				/opt/hostedtoolcache/go/1.24.12/x64/src/reflect/value.go:584
        	            				/opt/hostedtoolcache/go/1.24.12/x64/src/reflect/value.go:368
        	            				/home/runner/go/pkg/mod/github.com/bytedance/mockey@v1.2.14/internal/tool/call.go:36
        	            				/home/runner/go/pkg/mod/github.com/bytedance/mockey@v1.2.14/convey.go:43
        	            				/home/runner/go/pkg/mod/github.com/smartystreets/goconvey@v1.7.2/convey/discovery.go:89
        	            				/home/runner/go/pkg/mod/github.com/smartystreets/goconvey@v1.7.2/convey/context.go:279
        	            				/home/runner/go/pkg/mod/github.com/smartystreets/goconvey@v1.7.2/convey/context.go:112
        	            				/home/runner/go/pkg/mod/github.com/jtolds/gls@v4.20.0+incompatible/context.go:97
        	            				/home/runner/go/pkg/mod/github.com/jtolds/gls@v4.20.0+incompatible/gid.go:24
        	            				/home/runner/go/pkg/mod/github.com/jtolds/gls@v4.20.0+incompatible/stack_tags.go:108
        	            				/home/runner/go/pkg/mod/github.com/jtolds/gls@v4.20.0+incompatible/stack_tags.go:56
        	            				/home/runner/go/pkg/mod/github.com/jtolds/gls@v4.20.0+incompatible/stack_tags.go:49
        	            				/home/runner/go/pkg/mod/github.com/jtolds/gls@v4.20.0+incompatible/gid.go:24
        	            				/home/runner/go/pkg/mod/github.com/jtolds/gls@v4.20.0+incompatible/context.go:63
        	            				/home/runner/go/pkg/mod/github.com/smartystreets/goconvey@v1.7.2/convey/context.go:107
        	            				/home/runner/go/pkg/mod/github.com/smartystreets/goconvey@v1.7.2/convey/doc.go:76
        	            				/home/runner/go/pkg/mod/github.com/bytedance/mockey@v1.2.14/convey.go:48
        	            				/home/runner/work/juicefs/juicefs/pkg/chunk/disk_cache_test.go:632
        	Error:      	Not equal: 
        	            	expected: 0x6798c6c0
        	            	actual  : 0x6798bfb8
        	Test:       	TestCooldownAtimeOnWriteFixedOnLoad



1 total assertion

--- FAIL: TestCooldownAtimeOnWriteFixedOnLoad (0.00s)
FAIL
```

--- a/pkg/chunk/disk_cache_test.go
+++ b/pkg/chunk/disk_cache_test.go
@@ -623,13 +623,14 @@ func TestCooldownAtimeOnWriteFixedOnLoad(t *testing.T) {
 	dir := t.TempDir()
 	conf := defaultConf
 	conf.CacheExpire = time.Hour
+	conf.CacheEviction = EvictionNone
 	m := new(cacheManagerMetrics)
 	m.initMetrics()
 	cache := newCacheStore(m, dir, 1<<30, 1000, 1, &conf, nil)
 	key := "0_0_4"
 
-	fixedTime := time.Date(2025, 1, 28, 12, 0, 0, 0, time.UTC)
 	PatchConvey("mock time.Now to avoid drift", t, func() {
+		fixedTime := time.Date(2025, 1, 28, 12, 0, 0, 0, time.UTC)
 		Mock(time.Now).Return(fixedTime).Build()
 		path, err := cache.stage(key, []byte("test"))
 		require.NoError(t, err)

